### PR TITLE
Fixes #212

### LINKED ISSUE #212: Attempting to add a project while not authenticated should return a 401
Not sure what the exact code it throws is but I'm pretty sure it will throw a 403. Whatever the create a project endpoint throws to a non logged in user is the code we want to test that keeps being given.

### PR DESCRIPTION
**Pull request by zmike808**
Fixes #212

### CODE REVIEW

File: server/api/project/project.spec.js [hunk 1]
```diff
@@ -10,129 +10,145 @@ var agent = request.agent(app);
...
+describe('GET /api/projects', function () {
 
-  it('should respond with JSON array', function(done) {
+  it('should respond with JSON array', function (done) {
     request(app)
       .get('/api/projects')
       .expect(200)
       .expect('Content-Type', /json/)
-      .end(function(err, res) {
+      .end(function (err, res) {
```
Comment (agundy) [inline server/api/project/project.spec.js]: why the space here? We generally never put spaces here

Comment (zmike808) [inline server/api/project/project.spec.js]: Think that webstorm did that when I did an auto format for the file. Is there a link to our coding style guidelines?

Comment (kmcnellis) [inline server/api/project/project.spec.js]: https://github.com/rcos/Observatory3/blob/master/docs/CODINGSTANDARDS.md

Comment (kmcnellis) [inline server/api/project/project.spec.js]: Looks like the space is fine

**Pull request merged**